lib/deprecation_toolkit/rspec_hook.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "deprecation_toolkit/rspec"
+
+if defined?(RSpec)
```

Comment:
We are already doing that check when loading the file https://github.com/Shopify/deprecation_toolkit/pull/37/files#diff-bcf2a4ff97f78fe157315e9137464fc8R33